_posts/2017/2017-09-05-my-journey-to-become-a-developer.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: My journey to become a developer
+date: "2017-09-05 10:00:00"
+image: blog-header/journey-developer.jpg
+image-position: bottom
+categories: working-at-epages
+authors: ["Christoph"]
+---
+My name is Christoph, and since the beginning of the year I am working as a frontend developer at ePages.
+I originally started at the company in August 2015 with an apprenticeship as a media designer.
+During that time I mostly did website maintenance, some designs in sketch app or photoshop, and the one or other bootstrap website.
+
+## What is my background?
+
+My background was a solid but expandable knowledge of HTML, CSS, and the most basic design principles which I gained in a two-year schooling as a screen designer.
+I learned how to write concepts, how to create screen designs, and how to implement them as a static website.
+Fun fact: before starting out there I did not even know what HTML and CSS was.
+
+## The first few months in the team
+
+Since the beginning of 2017, with the graduation of my apprenticeship in sight, I started out in Team 42 to get a glimpse at what the life of a developer looks like.
+I had a very warm welcome in an interesting small team of two developers and one product owner.
+They immediately introduced me to the product they were currently working on, which was the Sales Analytics App built in [Rails](http://rubyonrails.org/).
+Without knowing anything about Rails, or the underlying programming language [Ruby](https://www.ruby-lang.org/en/), I was nonetheless able to contribute with my knowledge of CSS respectively Sass, and give the product some styling.
+Apart from that I got a crash course in how developers work, for instance with Git, and what is important to keep in mind in order to become a good developer.
+Always staying up to date and being interested in the newest technologies for example was one of that.
+
+## Status Quo
+
+Now, roughly six month after starting in the team, one exhausting project which was the relaunch of our website [epages.com](https://www.epages.com/) and successful finishing my apprenticeship in June, I am still feeling very satisfied with where I am.
```

Comment:
I know exactly what you mean when you say "exhausting", but for a public post, this sounds a bit negative. Can you find another adjective ;-)